examples/cloud-auth/README.md

```diff
@@ -0,0 +1,29 @@
+# Cloud-auth
+
+Cloud-auth is the way that we use to load a pre-defined Amazon Cognito template in a project. Instead of make the same template for each new project, a package with the template was created and we simply export it in the project.

```

Comment:
"Instead of making the same template for each new project, @ttoss/auth provides a package with a predefined template that we can import." sounds better.

Reply:
Done.